src/pages/mobile/NameTransfer.vue

```diff
@@ -0,0 +1,160 @@
+        @submit.prevent="transfer"
+      >
+        <AeInputAccount
+          v-model="accountTo"
+          v-validate="'required|account'"
+          autofocus
+          :error="errors.has('accountTo')"
+          :footer="errors.first('accountTo')"
+          name="accountTo"
+          :header="$t('transfer.send.to.recipient')"
+        />
+      </form>
+    </template>
+
+    <AeButton
+      :disabled="busy || errors.any()"
+      :form="_uid"
+      fill="secondary"
+      @click="transfer"
+    >
+      <AeLoader v-if="busy" /> {{ $t('next') }}
+    </AeButton>
+
+    <template v-if="!busy && inactiveAccounts.length">
+      <div class="own-account">
+        {{ $t('name.transfer.subaccount') }}
+      </div>
+      <ListItemAccount
+        v-for="account in inactiveAccounts"
+        :key="account.address"
+        border-dark
+        v-bind="account"
+        @click="transferToAccount(account.address)"
+      >
+        <LeftMore slot="right" />
+      </ListItemAccount>
+    </template>
+  </MobilePage>
+</template>
+
+<script>
+import { pick } from 'lodash-es';
+import { mapState, mapGetters } from 'vuex';
+import { handleUnknownError } from '../../lib/utils';
+import MobilePage from '../../components/mobile/Page.vue';
+import Guide from '../../components/Guide.vue';
+import AccountInline from '../../components/AccountInline.vue';
+import AeInputAccount from '../../components/AeInputAccount.vue';
+import AeButton from '../../components/AeButton.vue';
+import AeLoader from '../../components/AeLoader.vue';
+import ListItemAccount from '../../components/ListItemAccount.vue';
+import { LeftMore } from '../../components/icons';
+
+export default {
+  components: {
+    MobilePage,
+    Guide,
+    AccountInline,
+    AeInputAccount,
+    AeButton,
+    AeLoader,
+    ListItemAccount,
+    LeftMore,
+  },
+  props: {
+    name: { type: String, required: true },
+  },
+  data: () => ({ accountTo: '', busy: false }),
+  computed: {
+    ...mapState('names', {
+      nameRecord({ owned }) {
+        return owned.find(({ name }) => name === this.name);
+      },
+    }),
+    ...mapGetters('accounts', { activeAccount: 'active', activeColor: 'activeColor' }),
+  },
+  subscriptions() {
+    return pick(this.$store.state.observables, ['inactiveAccounts']);
+  },
+  mounted() {
+    const initialAccountIdx = this.$store.state.accounts.activeIdx;
+    const requredAccountIdx = this.$store.state.accounts.list
+      .findIndex(({ address }) => address === this.nameRecord.owner);
+    if (initialAccountIdx !== requredAccountIdx) {
+      this.$store.commit('accounts/setActiveIdx', requredAccountIdx);
+      this.$once('hook:destroyed', () => this.$store
+        .commit('accounts/setActiveIdx', initialAccountIdx));
+    }
+  },
+  methods: {
+    transferToAccount(address) {
+      this.accountTo = address;
+      this.transfer();
+    },
+    async transfer() {
+      if (!await this.$validator.validateAll()) return;
+      this.busy = true;
```

Comment:
I think before that line also should be dispatched confirm modal, if a user is trying to transfer to his own account.